src/network/networkctl.c

```diff
@@ -11,6 +11,7 @@
 #include <linux/if_bridge.h>
 #include <linux/if_tunnel.h>
 
+#include "json.h"
```

Comment:
we usually keep `sd-*.h` includes up top (since htey are our own "public" APIs) and move the internal ones down. This is an internal one, hence move it to the others down.